### PR TITLE
Add GPU based tensorflow job example

### PR DESCRIPTION
**Pull request by TommyLike**
See above.

### CODE REVIEW

File: example/integrations/tensorflow/Dockerfile.cpu [hunk 1]
```diff
@@ -9,5 +9,5 @@ RUN  apt-get update --fix-missing \
 && apt-get install -y git \
 && apt-get clean \
 && rm -rf /var/lib/apt/lists/*
-RUN pip install tf-nightly-gpu \
+RUN pip install tf-nightly \
```
Comment (k82cn) [inline example/integrations/tensorflow/Dockerfile.cpu]: if we're not using this dockerfile  in the yaml file, please move this to different director.

Comment (TommyLike) [reply example/integrations/tensorflow/Dockerfile.cpu]: hmm, this is a dockerfile that can actually work with the latest master branch.

Comment (k82cn) [reply example/integrations/tensorflow/Dockerfile.cpu]: if `volcanosh/example-tf:0.0.1` is not built from this dockerfile, please move it out and give a new name to `volcanosh/example-tf:0.0.1`.

Comment (TommyLike) [reply example/integrations/tensorflow/Dockerfile.cpu]: New images now:
1. example-tf-cpu: 0.0.1
2. example-tf-gpu: 0.0.1

Comment (k82cn) [reply example/integrations/tensorflow/Dockerfile.cpu]: As discussed before, this dockerfile is an example; and `example-tf:xxx` is retag from tensorflow-benchmark. So we need to make this message clear to others.

Comment (TommyLike) [reply example/integrations/tensorflow/Dockerfile.cpu]: `example-tf:xxx` is only used by OpenLab now, and those can be removed in the future, we don't use `example-tf:xxx` image

Comment (k82cn) [reply example/integrations/tensorflow/Dockerfile.cpu]: hm.. I'm confused about current status :( please make sure dockerfile & image in yaml align, all the time.

Comment (TommyLike) [reply example/integrations/tensorflow/Dockerfile.cpu]: @k82cn , ha let me summarize the history.

1. We used to have `example-tf.yaml` and `dockerfile`, in which `example-tf:0.0.1` image is used. But now they are all have been removed.
2. `dockerfile.cpu` is used to generate` example-tf-cpu.0.0.1` image, which is been used in `tf-example-cpu.yaml`
3. there is another `dockerfile.gpu` is used to generate` example-tf-gpu.0.0.1` image, and which is been used in `tf-example-gpu.yaml`

4. openlab generated another image example-tf:0.0.3 which is used for compare test with kubeflow.

Comment (k82cn) [reply example/integrations/tensorflow/Dockerfile.cpu]: > openlab generated another image example-tf:0.0.3 which is used for compare test with kubeflow.

do not use the same name of image; that's a really long history ...

Comment (TommyLike) [reply example/integrations/tensorflow/Dockerfile.cpu]: > > openlab generated another image example-tf:0.0.3 which is used for compare test with kubeflow.
> 
> do not use the same name of image; that's a really long history ...

Will be renamed when this cycle of tests finished.

File: example/integrations/tensorflow/Dockerfile.gpu [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# NOTE: the build process would change during developing.
+
+FROM nvidia/cuda:10.0-runtime-ubuntu16.04
+MAINTAINER volcano <maintainer@volcano.sh>
+RUN  apt-get update --fix-missing \
+&& apt-get install -y git python2.7 python-pip libcudnn7=7.4.1.5-1+cuda10.0 libcudnn7-dev=7.4.1.5-1+cuda10.0 \
+&& apt-get clean \
+&& rm -rf /var/lib/apt/lists/*
+RUN pip install tf-nightly-gpu \
```
Comment (lminzhw) [inline example/integrations/tensorflow/Dockerfile.gpu]: `tf-nightly-gpu` in pip and master branch of tensorflow/benchmarks may change rapidly.

I think we can use a special version/branch of tensorflow and benchmark to make the image more reliable and more reproducible.

Comment (TommyLike) [reply example/integrations/tensorflow/Dockerfile.gpu]: personally I prefer to use the package without specifying version unless the latest version has some issues/bugs. And for the master issue, I think we need to add a timestamp to the image tag.

Comment (lminzhw) [reply example/integrations/tensorflow/Dockerfile.gpu]: I think so, we should support at lease one of the timestamp or version/branch.

**Pull request closed without merging**